source/gui/settingsDialogs.py

```diff
@@ -1109,6 +1109,24 @@ def makeSettings(self, settingsSizer):
 		self.bindHelpEvent("SelectSynthesizerSynthesizer", self.synthList)
 		self.updateSynthesizerList()
 
+		# Translators: This is the label for the select output
+		# device combo in the synthesizer dialog. Examples of
+		# of an output device are default soundcard, usb
+		# headphones, etc.
+		deviceListLabelText = _("Audio output  &device:")
+		deviceNames=nvwave.getOutputDeviceNames()
+		# #11349: On Windows 10 20H1 and 20H2, Microsoft Sound Mapper returns an empty string.
+		if deviceNames[0] in ("", "Microsoft Sound Mapper"):
+			# Translators: name for default (Microsoft Sound Mapper) audio output device.
+			deviceNames[0] = _("Microsoft Sound Mapper")
+		self.deviceList = settingsSizerHelper.addLabeledControl(deviceListLabelText, wx.Choice, choices=deviceNames)
+		self.bindHelpEvent("SelectSynthesizerOutputDevice", self.deviceList)
+		try:
+			selection = deviceNames.index(config.conf["speech"]["outputDevice"])
+		except ValueError:
+			selection = 0
+		self.deviceList.SetSelection(selection)

```

Comment:
I think a single release cycle is enough time to transfer knowledge of the location of this.
Could you also please create a helper function to remove code duplication?
Please use the code from the audio panel as it has been linted/improved.
can you also wrap this in the following if statement?


```py
if version_year < 2024:
	_AddAudioOutput(self, sHelper)
```